packages/dashboard/src/components/playground/playground.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { ReactNode, useState } from 'react';
+import PropertiesPanelIcon from '~/components/resizablePanes/assets/propertiesPane.svg';
+import { CollapsiblePanel } from '~/components/internalDashboard/collapsiblePanel';
+import {
+  CONFIG_PANEL_COLLAPSE_WIDTH,
+  CONFIG_PANEL_OPEN_WIDTH,
+} from '~/components/playground/constants';
+import './playground.css';
+import { PropertyDrawer } from '~/components/playground/propertyDrawer/propertyDrawer';
+
+export const Playground = ({
```

Comment:
I'm confused by the naming of playground. Is there a more descriptive name we can use?

Reply:
I wanted a less confusing name than everything either has panel/panes or dashbaord in them. but yes, this will be changed